tests/bootstrap.php

```diff
@@ -1,4 +1,4 @@
 <?php
 
-$loader = require_once __DIR__ . "/../vendor/autoload.php";
-$loader->add('DocBlockReader\\', __DIR__);
\ No newline at end of file
+$loader = require __DIR__ . "/../vendor/autoload.php";
```

Comment:
I run the unit tests with `./vendor/bin/phpunit` and using `require_once` throws and error